arkane/statmech.py

```diff
@@ -643,126 +648,182 @@ def applyEnergyCorrections(E0, modelChemistry, atoms, bonds,
 
     if applyAtomEnergyCorrections:
         # Spin orbit correction (SOC) in Hartrees
-        # Values taken from ref 22 of http://dx.doi.org/10.1063/1.477794 and converted to hartrees
-        # Values in millihartree are also available (with fewer significant figures) from table VII of http://dx.doi.org/10.1063/1.473182
-        # Iodine SOC calculated as a weighted average of the electronic spin splittings of the lowest energy state. The splittings are
-        # obtained from Huber, K.P.; Herzberg, G., Molecular Spectra and Molecular Structure. IV. Constants of Diatomic Molecules, Van Nostrand Reinhold Co., 1979
-        SOC = {'H': 0.0, 'N': 0.0, 'O': -0.000355, 'C': -0.000135, 'S': -0.000893, 'P': 0.0, 'I':-0.011547226,}
+        # Values taken from ref 22 of http://dx.doi.org/10.1063/1.477794 and converted to Hartrees
+        # Values in milli-Hartree are also available (with fewer significant figures) from table VII of
+        # http://dx.doi.org/10.1063/1.473182
+        # Iodine SOC calculated as a weighted average of the electronic spin splittings of the lowest energy state.
+        # The splittings are obtained from Huber, K.P.; Herzberg, G., Molecular Spectra and Molecular Structure. IV.
+        # Constants of Diatomic Molecules, Van Nostrand Reinhold Co., 1979
+        SOC = {'H': 0.0, 'N': 0.0, 'O': -0.000355, 'C': -0.000135, 'S': -0.000893, 'P': 0.0, 'I': -0.011547226, }
 
         # Step 1: Reference all energies to a model chemistry-independent basis
         # by subtracting out that model chemistry's atomic energies
         # All model chemistries here should be lower-case because the user input is changed to lower-case
         if atomEnergies is None:
-            # Note: If your model chemistry does not include spin orbit coupling, you should add the corrections to the energies here
-            if modelChemistry.startswith('cbs-qb3'):  # only check start of string to allow different bond corrections (see below)
-                atomEnergies = {'H':-0.499818 + SOC['H'], 'N':-54.520543 + SOC['N'], 'O':-74.987624+ SOC['O'], 'C':-37.785385+ SOC['C'], 'P':-340.817186+ SOC['P'], 'S': -397.657360+ SOC['S']}
+            # Note: If your model chemistry does not include spin orbit coupling, you should add the corrections
+            # to the energies here
+            if modelChemistry.startswith(
+                    'cbs-qb3'):  # only check start of string to allow different bond corrections (see below)
```

Comment:
I would prefer putting the comment on the following line instead of splitting the code.